pkgs/applications/misc/procmail/default.nix

```diff
@@ -1,22 +1,20 @@
-{ lib, stdenv, fetchurl }:
+{ lib, stdenv, fetchurl, fetchFromGitHub, writeText, bash, msmtp }:
```

Comment:
`bash, ` seems unused

Reply:
@ajs124 this is correct. I also removed `writeText` which was unused as well.